common/persistence/versionHistory.go

```diff
@@ -23,83 +23,70 @@ package persistence
 import (
 	"fmt"
 	"github.com/uber/cadence/.gen/go/shared"
+	"github.com/uber/cadence/common"
 	"reflect"
 )
 
-type (
```

Comment:
i think it is ok to leave these definitions here
data interface file is too large

Reply:
I will leave the structs here as it will be easier for refactor.